components/org.wso2.carbon.identity.api.server.dcr/src/gen/java/org/wso2/carbon/identity/oauth2/dcr/endpoint/dto/ApplicationDTO.java

```diff
@@ -23,7 +23,7 @@
   private String clientSecret = null;
   
   
-  private String clientSecretExpiresAt = null;
+  private long clientSecretExpiresAt;
```

Comment:
since this is an autogenerated class, the swagger definition (https://github.com/wso2-extensions/identity-inbound-auth-oauth/blob/master/components/org.wso2.carbon.identity.api.server.dcr/src/main/resources/api.identity.oauth.dcr.endpoint.yaml#L346) should be updated and better to regenerate the code.

Reply:
This issue has been fixed with https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/1747/commits/515db3b42ca19cd16e0e55a82a910c10cb779dae